opentelemetry-zpages/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog
 
-## vNext
+## Unreleased
+
+### Changed
```

Comment:
0.7.0 is repeated. Also both api and sdk versions are bumped?

Reply:
Good catch. Fixed both.